proto/nftmarket/nftmarket.proto

```diff
@@ -43,18 +43,14 @@ message ListedNft {
 }
 
 message NftListing {
-  NftIdentifier nft_id       = 1 [(gogoproto.nullable) = false];
-  string        owner        = 2;
-  ListingType   listing_type = 3;
-  ListingState  state        = 4;
-  string        bid_token    = 5;
-  string        min_bid      = 6 [
-    (gogoproto.moretags)   = "yaml:\"min_bid\"",
-    (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Int",
-    (gogoproto.nullable)   = false
-  ];
-  uint64                    bid_active_rank = 7;
-  google.protobuf.Timestamp started_at      = 8
+  NftIdentifier             nft_id               = 1 [(gogoproto.nullable) = false];
+  string                    owner                = 2;
+  ListingType               listing_type         = 3;
+  ListingState              state                = 4;
+  string                    bid_token            = 5;
+  uint64                    minimum_deposit_rate = 6;
+  bool                      automatic_overdraft  = 7;
```

Comment:
Can you share the meaning of automatic overdraft @mkXultra ?

Reply:
this case
https://github.com/UnUniFi/chain/blob/f116b8eb5e51caf7d930d8942b27c617419ddc1a/docs/architecture/adr-nft-market/05.UX.md?plain=1#L24

Reply:
Deposit is done from multiple bids, if it's returned to the bidder who borrowed, what would be the result of this action? Borrow move move to other bids?